cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/internal/replication/ReplicationListener.java

```diff
@@ -0,0 +1,75 @@
+package com.cloudant.sync.internal.replication;
+
+import com.cloudant.sync.event.Subscribe;
+import com.cloudant.sync.event.notifications.ReplicationCompleted;
+import com.cloudant.sync.event.notifications.ReplicationErrored;
+import com.cloudant.sync.replication.ReplicationPolicyManager;
+import com.cloudant.sync.replication.Replicator;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * This class is not intended as API, it is public for EventBus access only.
+ * API consumers should not call these methods. See #setReplicationsCompletedListener for more
```

Comment:
There are no comments on `setReplicationsCompletedListener` in this class to see. Maybe we should just remove this sentence, as even the comments on `ReplicationPolicyManager#setReplicationsCompletedListener` aren't that useful in aiding understanding about replication lifecycle events.